client/src/components/ProgressBar.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "../stylesheets/ProgressBar.scss";
+
+const ProgressBar = (props) => {
+  return (
+    <div className="progress--bar" data-test="component-progressbar">
+      <div className="progress--bar_name" style={{ color: `${props.color}` }}>
```

Comment:
Ideally if possible styling should be abstracted to css :)

Reply:
Thank you once again for the review. We will see if we can achieve that for the color that we are now styling inline.